.github/workflows/build.yml

```diff
@@ -327,9 +327,9 @@ jobs:
           extra_build_args: |
             GOLANG_VERSION=${{ needs.get-go-version.outputs.go-version }}
 
-  build-docker-ubi-redhat-registry:
-    name: Docker ${{ matrix.arch }} ${{ matrix.fips }} UBI build for RedHat Registry
-    needs: [get-product-version, get-go-version, build]
+  build-docker-ubi:
+    name: Docker ${{ matrix.arch }} ${{ matrix.fips }} UBI builds
+    needs: [get-product-version, build]
```

Comment:
I think we still need `get-go-version` here as a dependency due to the `extra_build_args` that relies on it